server.py

```
@@ -73,30 +108,32 @@ async def request(self, method: str, endpoint: str, data: dict = None) -> dict:
                 
             except httpx.HTTPStatusError as e:
                 if e.response.status_code == 401:
-                    raise Exception("Invalid API token. Check TimeCamp settings")
+                    raise ToolError("Invalid API token. Check TimeCamp settings")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)